extensions/amp-next-page/1.0/service.js

```diff
@@ -727,10 +730,7 @@ export class NextPageService {
    * @private
    */
   getSeparatorElement_(element) {
-    const providedSeparator = childElementByAttr(
-      element,
-      'amp-next-page-separator'
-    );
+    const providedSeparator = childElementByAttr(element, 'separator');
```

Comment:
I'd recommend keeping the original name here since `separator` might be generically used (?)

Reply:
That was my initial thought which is why I changed it to `amp-next-page-separator` (it was initially `separator` in `amp-next-page` 0.1), however the [pattern in amp](https://github.com/ampproject/amphtml/search?q=childElementByAttr&unscoped_q=childElementByAttr) seems like we use `amp-*` for public css selectors but not for attributes (e.g. `amp-list`'s `load-more`, `amp-lightbox`'s `placeholder`, and `amp-subscriptions`' `subscriptions-action`, etc.)